lib/index.ts

```diff
@@ -913,12 +913,19 @@ export class RedisAdapter extends Adapter {
         });
         return numSub;
       });
+    // node-redis
     } else if (typeof this.pubClient.pSubscribe === "function") {
-      return this.pubClient
+      if(Object.getPrototypeOf(Object.getPrototypeOf(this.pubClient)).constructor.name === 'RedisCluster') {
+        return this.pubClient
```

Comment:
Shouldn't we loop over all nodes? I think this will get the number of subscribers on the current server only.